release-notes/10.0/releases.json

```diff
@@ -1,1733 +1,1733 @@
-            "hash": "f052ba22d9737c5cc00035ec6484bd0210e53837b2bc07e6bac08627d00213a8aad225912fd163425961d8fdc4ee5ddce0e7cab577e81ba5c79b546ded58eb9f"
-          },
-          {
-            "name": "windowsdesktop-runtime-win-arm64.zip",
-            "rid": "win-arm64",
-            "url": "https://download.visualstudio.microsoft.com/download/pr/b29e97a3-cbfe-4d51-82d8-54ceee017b4e/86c2bfd608fb8fe450373755472820c0/windowsdesktop-runtime-10.0.0-preview.1.25080.4-win-arm64.zip",
-            "hash": "e47be4dee94ac62bcc43389c6e7542002c280c694bb3780b9e67d0255f8701f238ff76f81bb5604b4df3ade478dedcf53a8ef56d9a2f1547e2c23b862a17cb4f"
-          },
-          {
-            "name": "windowsdesktop-runtime-win-x64.exe",
-            "rid": "win-x64",
-            "url": "https://download.visualstudio.microsoft.com/download/pr/4c52ec3d-f015-4e36-91c3-038f81611100/feb2ec87f1e079e6bdc452e449fc33c2/windowsdesktop-runtime-10.0.0-preview.1.25080.4-win-x64.exe",
-            "hash": "796ac79ed66d15cb4f0151645f2e424a60d41d034f6ac647515a93390a9c7d814fe6691c6a209957a14a296f499f0912f744a3fde7e7cc47855f624b8801439b"
-          },
-          {
-            "name": "windowsdesktop-runtime-win-x64.zip",
-            "rid": "win-x64",
-            "url": "https://download.visualstudio.microsoft.com/download/pr/6066b369-1693-4140-aa15-d49315d23989/de0df6260e5ca78dd3a936b5d7b702d7/windowsdesktop-runtime-10.0.0-preview.1.25080.4-win-x64.zip",
-            "hash": "0b89790cda0d425771d2a8159bf4db768e5931892392af163ffd34fa494dce1dc22f025efb799a513b77e7a2172a5012bf9211b35d2a9e755bc39c326f487c15"
-          },
-          {
-            "name": "windowsdesktop-runtime-win-x86.exe",
-            "rid": "win-x86",
-            "url": "https://download.visualstudio.microsoft.com/download/pr/8e4dbe38-608c-4905-bf19-7463ec8bab22/90d87611a88e829aceea7a43f6b9209c/windowsdesktop-runtime-10.0.0-preview.1.25080.4-win-x86.exe",
-            "hash": "985a3919b9a31cc74e8a8f448800406ae008b6b512b08b081bc88eb595f3c1a0caff4c65e316023a4cb538d52d4daea440326cbdbc9004d99e2b4b602a78d684"
-          },
-          {
-            "name": "windowsdesktop-runtime-win-x86.zip",
-            "rid": "win-x86",
-            "url": "https://download.visualstudio.microsoft.com/download/pr/833281c4-19cb-449f-b15f-dc98f90e6960/d8b098c08ca4c3c9186662375dbcdf5d/windowsdesktop-runtime-10.0.0-preview.1.25080.4-win-x86.zip",
-            "hash": "608b95683025ca023714ef07c5c6006f34eb01f161517a51039e08210be9d36fcaa1508c03fa63758ffabc1f13de55b902ff0c5032a3753a0e555b84c9e1ad6e"
-          }
-        ]
-      }
-    }
-  ]
-}
+{
+  "channel-version": "10.0",
+  "latest-release": "10.0.0-preview.3",
+  "latest-release-date": "2025-04-10",
+  "latest-runtime": "10.0.0-preview.3.25171.5",
+  "latest-sdk": "10.0.100-preview.3.25201.16",
+  "support-phase": "preview",
+  "release-type": "lts",
+  "lifecycle-policy": "https://aka.ms/dotnetcoresupport",
+  "releases": [
```

Comment:
What is the diff here? Line endings? I was expecting only the "url" properties to change.

Reply:
Probably. I ran my tool on Ubuntu.